tests/correctness/ntt-pipeline/ntt-32.futil

```diff
@@ -0,0 +1,4167 @@
+// +----+-------------------------+-------------------------+-------------------------+--------------------------+--------------------------+
```

Comment:
Sorry for yet another back-and-forth about this. Can we commit smaller examples as tests? The use case of these test files is that they are *human-readable*. When a change occur, the person committing the change should actually be able to read the file and see if something unexpected happened. No one will ever read a 4,000 line file.

Reply:
Ah, I just saw `ntt-4` down below. My expectation with the correctness test is that you don't actually commit the FuTIL file. The frontend first generates the file and then runs it using `fud`. Committing the FuTIL files has the downside that correctness tests don't remain in sync with changes to the frontend.

Reply:
No problem. That makes sense. I just went with ntt-32 for the correctness test since that's what the original naive method had. I'm going to be on the road today but I'll fix this later. I presume a few smaller correctness tests is viable as well.

Reply:
Update: I've removed then ntt-32 and added ntt-16, ntt-8 tests for correctness, with their cycle counts. These are now generated directly from the python script using a temporary file.

 I've left the ntt-4 for the frontend generation test, since I think ntt-2 is a bit too trivial.

Let me know if I'm missing anything.